### PR TITLE
LA-86: Clean data in app storage after upload file

### PR DESCRIPTION
**Pull request by manhduc-nguyen**
Please review this [https://app.zenhub.com/workspaces/linshare-flutter-5fe04e3547c3510015c66915/issues/linagora/linshare-mobile-flutter-app/86undefined](https://app.zenhub.com/workspaces/linshare-flutter-5fe04e3547c3510015c66915/issues/linagora/linshare-mobile-flutter-app/86undefined)

### CODE REVIEW

File: ios/LinshareShareExtension/ShareViewController.swift [hunk 1]
```diff
@@ -20,6 +20,12 @@ class ShareViewController: SLComposeServiceViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        // Clean all files before sharing new file
+        let shareFolderPath = FileManager.default
+            .containerURL(forSecurityApplicationGroupIdentifier: "group.\(hostAppBundleIdentifier)")!
+        let _ = self.cleanAllFilesBeforeSharing(dstURL: shareFolderPath)
+        
+
```
Comment (hoangdat) [inline ios/LinshareShareExtension/ShareViewController.swift:28]: IMO, we should delete temp file after uploaded

Comment (manhduc-nguyen) [reply ios/LinshareShareExtension/ShareViewController.swift:28]: okay I ll find the way to do it via flutter side

Comment (manhduc-nguyen) [reply ios/LinshareShareExtension/ShareViewController.swift]: @hoangdat @imGok  please check again guys

File: lib/presentation/util/helper/file_helper.dart [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+//  the Additional Terms applicable to LinShare software.
+//
+
+import 'dart:io';
+
+import 'package:domain/domain.dart';
+
+class FileHelper {
+  void deleteFile(FileInfo fileInfo) async {
+    final file = File(fileInfo.filePath + fileInfo.fileName);
```
Comment (hoangdat) [inline lib/presentation/util/helper/file_helper.dart:39]: what happen if `fileInfo` is null?

Comment (manhduc-nguyen) [reply lib/presentation/util/helper/file_helper.dart]: done, pls check again

File: lib/presentation/manager/upload_and_share_file/upload_and_share_file_manager.dart [hunk 3]
```diff
@@ -161,6 +168,8 @@ class UploadShareFileManager {
 
   void _handleUploadFileSucceed(FileUploadSuccess uploadSuccess) {
     final fileState = _uploadingStateFiles.getElementByUploadTaskId(uploadSuccess.uploadTaskId);
+    _fileHelper.deleteFile(fileState.file);
```
Comment (hoangdat) [inline lib/presentation/manager/upload_and_share_file/upload_and_share_file_manager.dart:171]: srr, I mean how about `fileState` null also

Comment (manhduc-nguyen) [reply lib/presentation/manager/upload_and_share_file/upload_and_share_file_manager.dart]: done, pls check again

**Review by hoangdat — APPROVED** [review]

**Pull request merged**